purger/src/rubin/nublado/purger/purger.py

```diff
@@ -226,18 +225,7 @@ async def _perform_report(self) -> None:
         if self._plan is None:
             raise PlanNotReadyError("Cannot report: plan not ready")
         rpt_text = str(self._plan)
-        if self._config.alert_hook is not None:
-            rpt_msg = SlackTextBlock(
-                heading="Purge plan",
-                text=rpt_text,  # May be truncated
-            )
```

Comment:
Like we talked about:

* We don't want to send the reports to Slack
* The Slack notifucation machinery is not wired up yet anyway. It will be in a future PR.